From 3f948f22d9106a2b957d1d430b16a7c51ccfc067 Mon Sep 17 00:00:00 2001
From: "Maxin B. John" <maxin.john@enea.com>
Date: Tue, 16 Jun 2015 22:53:23 +0200
Subject: [PATCH] Added missing dependencies

Upstream-Status: Submitted [linux-rt-users]

The following targets missed dependency on librttest.a:
rt-migrate-test
hackbench

Signed-off-by: Jackie Huang <jackie.huang@windriver.com>
Signed-off-by: Maxin B. John <maxin.john@enea.com>
---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index a48e759..7a3f192 100644
--- a/Makefile
+++ b/Makefile
@@ -85,7 +85,7 @@ hwlatdetect:  src/hwlatdetect/hwlatdetect.py
 	chmod +x src/hwlatdetect/hwlatdetect.py
 	ln -s src/hwlatdetect/hwlatdetect.py hwlatdetect
 
-rt-migrate-test: rt-migrate-test.o
+rt-migrate-test: rt-migrate-test.o librttest.a
 	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $< $(LIBS)
 
 ptsematest: ptsematest.o librttest.a
@@ -106,7 +106,7 @@ sendme: sendme.o librttest.a
 pip_stress: pip_stress.o librttest.a
 	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $< $(LIBS) $(RTTESTLIB)
 
-hackbench: hackbench.o
+hackbench: hackbench.o librttest.a
 	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $< $(LIBS)
 
 librttest.a: rt-utils.o error.o rt-get_cpu.o rt-sched.o
-- 
1.9.1

